chaoscenter/graphql/server/pkg/chaoshub/handler/handler.go

```diff
@@ -23,7 +23,7 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-const DefaultPath = "/tmp/"
+const DefaultPath = "./tmp/"
```

Comment:
Can you explain why you changed this code, please?

Reply:
Indeed. The issue stemmed from the tmp/ directory going unnoticed; the code couldn't locate it. 

While one solution could be adding a tmp directory in the root, it seemed irrelevant for the rest of the code. 

Consequently, I opted to update it to ./tmp, making it relative to the chaos-hub directory.


Would love to discuss on this as I am not sure how apt the solution is.

Reply:
I wouldn't recommend changing production code that's currently working well just to make our test code pass. We don't inspect the potential side effects that could occur on that code.

Reply:
Yep, that's true. 
So should we close this PR and then, I'll comment out the failing test in a new PR?

Reply:
You can complete this in the current PR. I will create a new pull request to address the failing test cases(commented test cases).